README.md

```diff
@@ -1,6 +1,10 @@
-# Metrics API for [erldns](https://github.com/dnsimple/erldns)
+# Metrics API for [erldns]
```

Comment:
I see the link has been removed but we still keep the `[]` around was this intentional?

Reply:
Yes if you have a footnote in Markdown, like

```markdown
[erldns]: https://...
```

anywhere on the page, then you can just use `[erldns]` instead of `[erldns][erldns]` 🙃